### PR TITLE
Posts: Move all posts to yearly folders

### PR DESCRIPTION
**Pull request by SchoolGuy**
This is a splitout of #178. It should help keep track easier of the different posts that we have.

### CODE REVIEW

**Review by ByteOtter — APPROVED** [review]
lgtm 👍🏼

**Review by ByteOtter — APPROVED** [review]

**Review by tiltingpenguin — DISMISSED** [review]
LGTM

File: Gemfile [hunk 1]
```diff
@@ -1,2 +1,5 @@
 source 'https://rubygems.org'
-gem 'github-pages'
\ No newline at end of file
+gem 'github-pages'
+# Uncomment the following line for local development
+# See: https://github.com/envygeeks/jekyll-docker/issues/340
+# gem "webrick"
```
Comment (tiltingpenguin) [inline Gemfile:5]: Nitpick: I would change the order of the comments. Like this it isn't as obvious that you should uncomment the gem line

**Review by tiltingpenguin — APPROVED** [review]
Still LGTM

File: Gemfile (hunk 1 above)
Comment (SchoolGuy) [reply Gemfile:5]: Thanks. Will change that. :)

Comment (tiltingpenguin) [reply Gemfile]: Another nitpick: Also I just noticed that we use double quotes instead of single quotes here which is different from above

**Pull request merged**